libnethack/include/rtp.h

```diff
@@ -0,0 +1,42 @@
+#ifndef RTP_H
+#define RTP_H
+
+#include "extern.h"
+#include "hack.h"
+#include "monst.h"
+
+#define RTP_NAME_COUNT 16
+
+// These are added in order of when I got permission
+#define RTP_NAMES { \
+    "Liam Middlebrook", \
+    "Stephanie Miller", \
+    "McSaucy", \
+    "Russ Harmon", \
+    "Jordan Rodgers", \
+    "Ethan House", \
+    "Rob Glossop", \
+    "Grant Cohoe", \
+    "Steve Greene", \
+    "William Dignazio", \
+    "Chris Lockfort", \
+    "Will Orr", \
+    "Derek Gonyeo", \
+    "Kevin Thompson", \
+    "Angelo DiNardi", \
+    "Dan Willemsen", \
+}
+
+// This is a pretty crappy WAR, but for now it makes more sense than writing a
+// more elegant solution. Hopefully this will become a bigger problem in the
+// future
+
+// The expanded format for this should be something like ((i == 1) || (i == 17))
+#define FEMALE_RTPS(i) ((i == 1))
+
+struct monst *name_rtp(struct monst *mtmp);
+
+void player_killed_rtp(struct level *lev);
+
+struct obj *create_rtp_corpse(struct level *lev, int x, int y, enum rng rng);
+#endif
```

Comment:
Use a comment to demarcate which if statement this endif goes to. E.g.

#endif  // RTP_H